src/components/Header.css

```diff
@@ -0,0 +1,13 @@
+li.nav-item > a.selected {
+  background-color: #ff9800;
+  color: #ffffff;
```

Comment:
can you add 
  box-shadow: 0 4px 20px 0px rgba(0, 0, 0, 0.14), 0 7px 10px -5px rgba(255, 152, 0, 0.4);

Reply:
Thanks! added